GUIDE.md

```diff
@@ -160,4 +160,17 @@ nvm use 20
 ### Install snarkjs
 ```sh
 npm install -g snarkjs
-```
\ No newline at end of file
+```
+
```

Comment:
We may want to introduce a little what will the users are expected to go through after the es-node launch. For example, they will basically go through two phases:
1. Data sync phase: they can tell how many peers are connected, and more importantly, the estimated syncing time. we can only explain a little bit about why the CPUs are full, and how to tune it using p2p.sync.concurrency & p2p.fill-empty.concurrency. (Or move it to FAQ)
2. Sampling phase: they may need to find out whether the es-node can do enough sampling in one slot, and if the log shows " Mining tasks timed out", what should they do? Check the read IOPS of their disk, and they can also tune it by using --miner.threads-per-shard